test/infrastructure/docker/internal/docker/loadbalancer.go

```diff
@@ -161,16 +162,22 @@ func (s *LoadBalancer) UpdateConfiguration(ctx context.Context) error {
 			return errors.Wrapf(err, "failed to get IP for container %s", n.String())
 		}
 		if s.ipFamily == clusterv1.IPv6IPFamily {
-			backendServers[n.String()] = net.JoinHostPort(controlPlaneIPv6, "6443")
+			backendServers[n.String()] = controlPlaneIPv6
```

Comment:
This change seems unrelated, could you provide some context?

Reply:
I might be wrong but it seemed like a bug to me, we are passing `controlPlanePort` as part of `LoadBalancer` object, and on these lines, it's hardcoded to "6443". I removed it and the port is later processed in the template.

Reply:
I split the backend and frontend control plane port in the `Loadbalancer` object, with a custom template we might want the server address to contain only IP address without a port.
With this change, the port will be appended to the IP address later when processing the template.